webui/react/src/components/BreadcrumbBar.tsx

```diff
@@ -20,33 +23,26 @@ interface Props {
   project?: Project;
   trial?: TrialDetails;
   type: 'project' | 'experiment' | 'trial';
-  workspace?: Workspace;
+  workspaceId?: number;
 }
 
 const BreadcrumbBar: React.FC<Props> = ({
   id,
   type,
-  workspace: workspaceIn,
+  workspaceId,
   project: projectIn,
   experiment: experimentIn,
   trial: trialIn,
   extra,
 }: Props) => {
-  const [workspace, setWorkspace] = useState<Workspace | undefined>(workspaceIn);
   const [project, setProject] = useState<Project | undefined>(projectIn);
   const [experiment, setExperiment] = useState<ExperimentBase | undefined>(experimentIn);
   const [trial, setTrial] = useState<TrialDetails | undefined>(trialIn);
   const [canceler] = useState(new AbortController());
 
-  const fetchWorkspace = useCallback(async () => {
-    if (!project?.workspaceId) return;
-    try {
-      const response = await getWorkspace({ id: project.workspaceId }, { signal: canceler.signal });
-      setWorkspace(response);
-    } catch (e) {
-      handleError(e, { publicSubject: 'Unable to fetch workspace.' });
-    }
-  }, [canceler.signal, project?.workspaceId]);
+  const workspace = useObservable(
```

Comment:
**nit:** I would probably call this variable `loadableWorkspace` so it's a bit more obvious that it's a loadable instead of an actual workspace object.

Reply:
✔️